ios/RnLdk.swift

```diff
@@ -106,66 +106,76 @@ class MyFilter: Filter {
+        guard let cOpaqueStruct = keysInterface.cOpaqueStruct  else {
+            let error = NSError(domain: "start cOpaqueStruct failed", code: 1, userInfo: nil)
+            return reject("start", "Failed",  error)
+        }
+        _ = Bindings.LDKThirtyTwoBytes_to_array(nativeType: cOpaqueStruct.get_secure_random_bytes(cOpaqueStruct.this_arg))
+        let userConfig = UserConfig.init()
         
-        if (serializedChannelManagerHex != "") {
-            var serializedChannelMonitors: [[UInt8]] = [];
-            let hexesArr = monitorHexes.split(separator: ",");
+        if (!serializedChannelManagerHex.isEmpty) {
+            var serializedChannelMonitors: [[UInt8]] = []
+            let hexesArr = monitorHexes.split(separator: ",")
             for hex in hexesArr {
                 serializedChannelMonitors.append(hexStringToByteArray(String(hex)))
             }
             
-            let serialized_channel_manager: [UInt8] = hexStringToByteArray(serializedChannelManagerHex);
+            let serialized_channel_manager: [UInt8] = hexStringToByteArray(serializedChannelManagerHex)
             
             do {
-                channel_manager_constructor = try ChannelManagerConstructor(channel_manager_serialized: serialized_channel_manager, channel_monitors_serialized: serializedChannelMonitors, keys_interface: keysInterface, fee_estimator: feeEstimator, chain_monitor: chain_monitor!, filter: filter, router: nil, tx_broadcaster: broadcaster, logger: logger)
+                channel_manager_constructor = try ChannelManagerConstructor(channel_manager_serialized: serialized_channel_manager, channel_monitors_serialized: serializedChannelMonitors, keys_interface: keysInterface, fee_estimator: feeEstimator, chain_monitor: chain_monitor, filter: filter, router: nil, tx_broadcaster: broadcaster, logger: logger)
             } catch {
-                reject("start", "Failed",  error);
-                return;
+                reject("start", "Failed",  error)
+                return
             }
         } else {
-            channel_manager_constructor = ChannelManagerConstructor(network: LDKNetwork_Bitcoin, config: userConfig, current_blockchain_tip_hash: hexStringToByteArray(blockchainTipHashHex), current_blockchain_tip_height: UInt32(truncating: blockchainTipHeight), keys_interface: keysInterface, fee_estimator: feeEstimator, chain_monitor: chain_monitor!, router: nil, tx_broadcaster: broadcaster, logger: logger);
+            channel_manager_constructor = ChannelManagerConstructor(network: LDKNetwork_Bitcoin, config: userConfig, current_blockchain_tip_hash: hexStringToByteArray(blockchainTipHashHex), current_blockchain_tip_height: UInt32(truncating: blockchainTipHeight), keys_interface: keysInterface, fee_estimator: feeEstimator, chain_monitor: chain_monitor, router: nil, tx_broadcaster: broadcaster, logger: logger)
```

Comment:
how dod u manage to get rid of `!` here?  i think xcode bitches that it might be null or smth

Reply:
I used guard